### PR TITLE
Do not copy modular-info.class into resulting jar

### PR DESCRIPTION
**Pull request by fedinskiy**
Fix for #297

### CODE REVIEW

File: src/it/package-archive-exclude-module-info-it/pom.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~
+  ~   Copyright (c) 2016-2018 Red Hat, Inc.
```
Comment (tsegismont) [inline src/it/package-archive-exclude-module-info-it/pom.xml:4]:
```suggestion
  ~   Copyright (c) 2016-2021 Red Hat, Inc.
```

File: src/it/package-archive-exclude-module-info-it/src/main/java/org/vertx/demo/MainVerticle.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/*
+ *
+ *   Copyright (c) 2016-2018 Red Hat, Inc.
```
Comment (tsegismont) [inline src/it/package-archive-exclude-module-info-it/src/main/java/org/vertx/demo/MainVerticle.java:3]:
```suggestion
 *   Copyright (c) 2016-2021 Red Hat, Inc.
```

File: src/it/package-archive-exclude-module-info-it/verify.groovy [hunk 3]
```diff
@@ -0,0 +1,26 @@
+/*
+ *
+ *   Copyright (c) 2016-2018 Red Hat, Inc.
```
Comment (tsegismont) [inline src/it/package-archive-exclude-module-info-it/verify.groovy:3]:
```suggestion
 *   Copyright (c) 2016-2021 Red Hat, Inc.
```

**Review by tsegismont — APPROVED** [review]
This looks good to me, thank you @fedinskiy !

Just a few suggested changes regarding license headers.

**Comment by fedinskiy** [comment]
@tsegismont, copyright lines changed at your suggestion.

**Pull request merged**